[baking-with-ledger] handles feedback (#526)
trilitech/tezos-developer-docs: docs/tutorials/bake-with-ledger/run-baker.md

Complete the [Step 1: Run an Octez node](/tutorials/join-dal-baker/run-node) of the tutorial, and make following changes in [Step 2: Set up a baker account](/tutorials/join-dal-baker/prepare-account).

>  - If you **don't want to use a consensus key**, use your Ledger key directly as a baker. Import it from the `octez-signer` remote with the following command:

Reviewer: ```suggestion
 - You can use your Ledger key as your main baker key or you could use Ledger key as consensus key.
To use the Ledger key as your main baker key, use the Le. Import it from the `octez-signer` remote with the following command:
```

Reviewer: The point i am trying to convey to the user is that he can use ledger to sign baking operations while keeping the ledger key as main baker key or he can use the ledger key as consensus key. If he uses the Ledger as consensus key, he will have to use another main baker key for singing stake etc operations.

    ```bash
    octez-client import secret key my_baker remote:tz...
    ```

    > Replace the `tz...` with the public key hash of your Ledger baking key.

>     In that case, to be able to sign the operations required to set up your baker, you need to use the `Tezos Wallet (XTZ)` application.

Reviewer: ```suggestion
    Run and sign the following operations to set up your baker. You will need to use the `Tezos Wallet (XTZ)` application.
```

    Quit the `Tezos Baking` application and open the `Tezos Wallet (XTZ)` application. Then setup your baker.

    ```bash
    octez-client import secret key my_baker remote:tz...
    octez-client register key my_baker as delegate
    octez-client stake 6000 for my_baker
    ```

    Your baker account is now set up and ready to bake using the Ledger.

>  - If you **want to use a consensus key**, instead of generating it, use the Ledger key. To do so, import it from the `octez-signer` remote with the following command:

Reviewer: ```suggestion
 - If you **want to use your Ledger key as a consensus key**, import it from the `octez-signer` remote with the following command:
```

    ```bash
    octez-client import secret key consensus_key remote:tz...
    ```

    > Replace the `tz...` with the public key hash of your Ledger baking key.

>     You can then continue to set up your baker account.

Reviewer: ```suggestion
    With Ledger key imported as consensus key, you will need to generate/setup your baker key separately. You can then continue to set up your baker account. See the following commands:
```
